src/doc-printer.js

```diff
@@ -398,7 +398,12 @@ function printDocToString(doc, options) {
                     out.pop();
                   }
 
-                  if (out.length) {
+                  if (
```

Comment:
Is there precedent for using the notion of "parser" in `doc-*` files?

Reply:
No, but this is the easiest way to support double-space `break`.

That's why I chose the backslash style `break`, the trailing whitespace syntax is super weird and conflict with the `trim_trailing_whitespace` rule.